packages/admin-ui/src/pages/installer/components/Steps/Notifications.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+// Components
+import Card from "components/Card";
+import Button from "components/Button";
+import { CustomEndpoint, GatusEndpoint } from "@dappnode/types";
+import "./../../../notifications/tabs/Settings/settings.scss";
```

Comment:
it looks weird to use an import of a style from a far away directory. Could it be implemented somehow importing a component with the style applied already ?